.github/workflows/fuzzers.yml

```diff
@@ -36,7 +36,9 @@ jobs:
           - entropic
           - fairfuzz
           - fastcgs
+          - fastcgs_huge
```

Comment:
Do we need to test this feature with both? Or do you think one is enough to tell whether it is useful?

Reply:
One is actually enough to test this feature. I wanted to compare huge page versions of fastcgs and fastcgs_lm, that's why added both. Please let me know if I should remove one of them.

Reply:
Let's do one for now and if it does better we can try it on the variant.

Reply:
If you think we're likely to miss something by this approach (I don't know much about the big page tables and know nothing about fastcgs_lm) then we can keep. I'm just trying to cut back on unnecessary runs by multiplying features.

Reply:
Done. Kept fastcgs_lm_huge and removed fastcgs_huge.